### PR TITLE
Multi Namespace test PR

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

**Comment by smark88** [comment]
Debating on removing the travis.yaml it pretty much doesn't work now. @thathert @fboulnois

**Comment by smark88** [comment]
Calling this done SAM cannot handle more than 3+ concurrent tests.

File: .github/workflows/gradle-build-pr.yml [hunk 1]
```diff
@@ -96,16 +103,50 @@ jobs:
...
+                printf "Namespace ${i} not in use Deploying integration test to ${i}\n"
+                kubectl create secret generic ${i}-inuse --from-literal=inuse=${i} -n ${i}
+                tail=$(echo $i | awk -F- {'print $2'})
+                echo "::set-env name=IT_JADE_API_URL::https://jade-${tail}.datarepo-integration.broadinstitute.org"
+                echo "::set-env name=NAMESPACEINUSE::${i}"
+                return 0
+              fi
+            done
+            sleep 60
+            find_namespace
```
Comment (fboulnois) [inline .github/workflows/gradle-build-pr.yml:125]: Perhaps we might want to retry only a few times instead of infinitely?

**Review by fboulnois — CHANGES_REQUESTED** [review]
lgtm, consider rebasing into a set of logical commits and one other suggestion.

File: .github/workflows/gradle-build-pr.yml [hunk 2]
```diff
@@ -96,16 +103,50 @@ jobs:
...
+          DEPLOY_CHART_VERSION: 0.0.7
+        run: |
+          helm namespace upgrade ${NAMESPACEINUSE}-secrets datarepo-helm/create-secret-manager-secret --version=${SECRET_CHART_VERSION} \
+            --install --namespace ${NAMESPACEINUSE} -f \
+            "https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/integration/${NAMESPACEINUSE}/${NAMESPACEINUSE}Secrets.yaml"
+          helm namespace upgrade ${NAMESPACEINUSE}-jade datarepo-helm/datarepo --version=${DEPLOY_CHART_VERSION} --install \
+            --namespace ${NAMESPACEINUSE} -f \
+            "https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/integration/${NAMESPACEINUSE}/${NAMESPACEINUSE}Deployment.yaml" \
+            --set "datarepo-api.image.tag=${GCR_TAG}"
+      # build and push container to integration
```
Comment (fboulnois) [inline .github/workflows/gradle-build-pr.yml]: Also, I think this comment applies more to the "deploying new image to kubernetes" section

File: .github/workflows/gradle-build-pr.yml (hunk 1 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml:125]: It seems fine on infinite worst case the job is killed in 180 minutes.


This deploy was in queue for 30 minutes ish. 
https://github.com/DataBiosphere/jade-data-repo/runs/499537076?check_suite_focus=true

File: .github/workflows/gradle-build-pr.yml (hunk 2 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml]: deleted

**Review by thathert — APPROVED** [review]
LGTM

File: .github/workflows/gradle-build-pr.yml (hunk 1 above)
Comment (thathert) [reply .github/workflows/gradle-build-pr.yml:125]: Given that there's no way to retry or handle unaddressed events yet, I think we'll want to poll until Github kills the action after 3 hours.

**Pull request merged**